web-local/test/ui/utils/startRuntimeForEachTest.ts

```diff
@@ -50,13 +63,20 @@ export function startRuntimeForEachTest() {
   });
 
   test.afterEach(async () => {
-    const processExit = new Promise((resolve) => {
-      childProcess.on("exit", resolve);
```

Comment:
1. Might it be a problem that it was registering the `on` callback inside the promise, which will run after the kill signal has been sent instead of after? Might be better to register the callback maybe even when creating the process.
2. Based on some googling, it seems the `"close"` event would be safer than the `"exit"` event